microshift_install_rpm/microshift-install-rpm.adoc

```diff
@@ -18,7 +18,7 @@ include::modules/microshift-install-rpms.adoc[leveloffset=+1]
 [role="_additional-resources"]
 == Additional resources
 * xref:../microshift_install_get_ready/microshift-fips.adoc#microshift-fips[Using FIPS mode with {microshift-short}]
-* Download the link:https://console.redhat.com/openshift/install/pull-secret[pull secret] from the Red Hat Hybrid Cloud Console
```

Comment:
Please do not delete this text, it is essential and will need addressed in CQAs.